frontend/pages/hubs.js

```diff
@@ -1,9 +1,10 @@
 import React from "react";
+import axios from "axios";
+import { Typography, Container, makeStyles, useMediaQuery } from "@material-ui/core";
+
```

Comment:
Nit: I like to group 3rd party deps at the top of the file, and then local files below them. I find this makes it easier to grok a file. Maybe a good addition to a "codebase health and style" guide? Let me know if this doesn't align with with anyone's preferences and happy to not do it 🙏

Reply:
I think this makes a lot of sense :+1: